service/db.py

```diff
@@ -17,41 +18,57 @@
 session = Session()
 
 
-def read_data_for_new_task() -> Dict[str, Any]:
-    next_task = session.query(func.min(Task.id)).filter(
+def read_data_for_new_task(user) -> Dict[str, Any]:
+    resp = session.query(func.min(Task.id)).filter(
         Task.task_status_id == 1
     ).scalar()
-    next: Task = session.query(Task).filter(Task.id == next_task).all()[0]
-    result = {
-        'latitude': next.latitude,
-        'longitude': next.longitude
-    }
+    next_task: Task = session.query(Task).filter(Task.id == resp).all()[0]
+    if user == 'admin':
+        result = TaskToAdmin.from_orm(next_task)
+    if user == 'drone':
+        result = TaskToDrone.from_orm(next_task)
+    if user != 'admin' or 'drone':
+        result = 'Not enough rights'
     return result
 
 
-def add_coordinate_to_db(add_latitude, add_longitude):
-    last_coord_id = session.query(func.max(Task.id)).scalar()
-    new_coord = Task(
-        id=last_coord_id+1,
-        latitude=add_latitude,
-        longitude=add_longitude,
-        task_status_id=1
-    )
-    session.add_all([new_coord])
-    session.commit()
-    print('\n Data entered \n')
+def add_task_to_db(user, add_latitude, add_longitude, add_priority):
+    if user == 'user' or 'admin':
+        last_coord_id = session.query(func.max(Task.id)).scalar()
+        new_coord = Task(
+            id=last_coord_id+1,
+            latitude=add_latitude,
+            longitude=add_longitude,
+            task_status_id=1,
+            priority=add_priority
+        )
+        session.add_all([new_coord])
+        session.commit()
+        result = (
+            f'The data is entered.\n'
+            f'Latitude is {add_latitude}\n'
+            f'Longitude is {add_longitude}\n'
+            f'Priority: {add_priority}\n'
+        )
+    else:
+        result = 'Not enough rights'
+    return result
 
 
-def add_dron_status(drone_id, battery, departure_lat, departure_long):
-    new_info = DroneStatus(
-        id=drone_id,
-        battery_charge_lvl=battery,
-        departure_latitude=departure_lat,
-        departure_longitude=departure_long
-    )
-    session.add_all([new_info])
-    session.commit()
-    print('\n Information about the drone has been received \n')
+def add_dron_status(user, drone_id, battery, departure_lat, departure_long):
+    if user == 'DB' or 'admin':
+        new_info = DroneStatus(
+            id=drone_id,
+            battery_charge_lvl=battery,
+            departure_latitude=departure_lat,
+            departure_longitude=departure_long
+        )
+        session.add_all([new_info])
+        session.commit()
+        result = 'Information about the drone has been received\n'
+    else:
+        result = 'The database updates the data automatically'
```

Comment:
If I am not an Admin - I can call this endpoint. Will this endpoint update the data automatically?